Automattic/jetpack
# Update Jetpack Social media picker UI to match new designs

This PR implements the new design for the media picker UI in Jetpack Social.

## Proposed changes:

* Update media picker to match new design.
* Add option to remove picked media.
* Update error messages to be more descriptive.

### Other information:


## Jetpack product discussion


p1HpG7-jqQ-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Add support for enhanced publishing to your site (context here: D90213-code)
* Start a new post.
* Make sure the design matches the screenshots.
* Check that you can pick an image.
* Check that you can replace an image by clicking on the current on.
* Check that you can remove an image.
* Make sure the tests pass: `cd projects/js-packages/publicize-components && npm test src/components/media-section/test/index.test.js`

<img width="267" alt="image" src="https://user-images.githubusercontent.com/1713699/214388276-b8d1507b-ca91-4a5e-8de5-38214704f89b.png">
<img width="272" alt="image" src="https://user-images.githubusercontent.com/1713699/214388310-96199b91-0aff-4f16-8ddb-6b819cd6b9da.png">
